README.md

```diff
@@ -3,115 +3,94 @@
 ![Coverage badge](https://img.shields.io/endpoint.svg?url=https%3A%2F%2Fsilent-zebra-36.tunnel.datahub.at%2Fcoverage%2Frigbox%2Fmaster)
 ![Build status badge](https://img.shields.io/endpoint.svg?url=https%3A%2F%2Fsilent-zebra-36.tunnel.datahub.at%2Fstatus%2Frigbox%2Fmaster)
 
-Rigbox is a high-performance, open-source MATLAB toolbox for managing behavioral neuroscience experiments. Initially developed to probe mouse behavior for the [Steering Wheel Setup](https://www.ucl.ac.uk/cortexlab/tools/wheel),  Rigbox simplifies hardware/software interfacing and creates a runtime environment in which an experiment's parameters can be easily monitored and manipulated.
+Rigbox is a high-performance, open-source MATLAB toolbox for managing behavioral neuroscience experiments. Initially developed to probe mouse behavior for the [Steering Wheel Setup](https://www.ucl.ac.uk/cortexlab/tools/wheel), Rigbox is under active, test-driven development to encompass running experiments across a variety of experimental paradigms in behavioral neuroscience.
 
-Rigbox includes many features including synchronizing recordings, managing experimental data and a viewing model for visual stimuli.
-
-Rigbox is mostly object-oriented and highly modular, making designing new experiments much simpler. Rigbox is currently under active, test-driven development. 
+Rigbox's main goals are to simplify hardware/software interfacing, behavioral task design, and visual and auditory stimuli presentation. Additionally, Rigbox can time-align datastreams from multiple sources and communicate with a remote database to manage experiment data. Rigbox is mostly object-oriented and highly modular, which simplifies the process of designing experiments.
 
 ## Requirements
 
-For running full experiments Rigbox requires two PCs: one for presenting stimuli and one for monitoring the experiment.  Currently only National Instruments DAQs are supported for acquiring data from hardware devices.  For testing, the toolbox can be run on a single machine.  
+For exploring Rigbox's features and running test experiments, Rigbox only needs to be installed on a single computer.
+
+For running experiments, we recommend installing Rigbox on two computers: one computer (which we refer to as the "Stimulus Computer" or "SC") communicates with an experiment rig's hardware and presents stimuli, and the other computer (which we refer to as the "Master Computer" or "MC") runs a GUI that the experimenter can use to start, monitor, parameterize, and stop the experiment.
```

Comment:
yeah, we'll disagree on this. I think they're helpful, and they are used immediately below

Reply:
This is really about rigbox having two components: "MC" and "SC". These can be run on the same on separate machines. For debuggin, these can run on the same machine. For actual use, it is recommended that each component is ran on separate machines.

Reply:
@anwarnunez  how do you feel about using the terms 'MC' and 'SC' here? I'm in favor of it because I refer to them multiple times later in the readme in the "Requirements" and "Getting Started" sections, but Miles is against it.